test/npm-shrinkwrap.js

```diff
@@ -2,12 +2,27 @@ var test = require('tape');
 var fixtures = require('fixtures-fs');
 var path = require('path');
 var fs = require('fs');
+var mkdirp = require("mkdirp");
+
+// a hack to workaround fixtures-fs
+// override fs.mkdirp and leaving all other functions intact
+var _fs = Object.create(fs);
+_fs.mkdir = mkdirp;
```

Comment:
`mkdirp` is used to create folders like `@th507/foo` in one stroke.
 
In `fixtures-fs` package, `create-fixtures.js` line 28, it uses `opts.fs.mkdir` instead of `opts.mkdirp` to make directory.

When I wish to override `mkdir` behavior, I have to provide a fake `opts.fs`, not `opts.mkdirp`. 
I tried `opts.fs = { mkdir: function(){...} }`, but it failed because the script required other function/property from `opts.fs` object.

I did tried to patch `create-fixtures.js`. However, simply replacing `fs.mkdir` with `mkdirp` broke one of the unit tests, and there was not simpler solution in sight.

I could work on `fixtures-fs` package later and then remove the hack.
But I think maybe we could focus on bringing (basic) scoped package support `npm-shrinkwrap` for the moment.